server/src/infra/entities/user.entity.ts

```diff
@@ -42,6 +42,9 @@ export class UserEntity {
   @Column({ default: '' })
   profileImagePath!: string;
 
+  @Column({ default: '' })
```

Comment:
You will have to generate a database migration for the newly added column

Reply:
Please see https://immich.app/docs/developer/database-migrations

Reply:
I reverted the changes regarding the user entity, as it is not needed. I think the database migration is not needed then.